vulture_os/system/config/views.py

```diff
@@ -107,10 +114,14 @@ def render_form(**kwargs):
         if "internal_tenants" in form.changed_data:
             Cluster.api_request("services.rsyslogd.rsyslog.configure_pstats")
             Cluster.api_request("services.rsyslogd.rsyslog.restart_service")
-        if "portal_cookie_name" in form.changed_data:
-            # Reload Frontends with authenticated Workflows: session checks must be updated with the new cookie's name
-            for frontend in Frontend.objects.filter(workflow__authentication__isnull=False).distinct():
-                frontend.reload_conf()
+        if "portal_cookie_name" or "public_token" in form.changed_data:
```

Comment:
```suggestion
        if "portal_cookie_name" or "public_token" in form.changed_data:
```
Nope, doesn't work like that!

Reply:
You are absolutely right on that ! Thank you